### PR TITLE
#2 :green_heart: update OCI configuration to get helm chart publishing

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**

### CODE REVIEW

File: extensions/extensions-helm/pom.xml [hunk 1]
```diff
@@ -105,18 +110,39 @@
...
-                        <skipInit>true</skipInit>
                         <appVersion>${project.version}</appVersion>
                         <chartDirectory>${project.basedir}</chartDirectory>
                         <namespace>default</namespace>
                         <templateGenerateName>true</templateGenerateName>
                         <skip>${skip.helm}</skip>
-                        <skipTemplate>false</skipTemplate>
                         <skipDependencyBuild>true</skipDependencyBuild>
+                        <!-- For OCI, do not init: -->
+                        <skipInit>false</skipInit>
```
Comment (ewilkins-csi) [inline extensions/extensions-helm/pom.xml:140]: Q: There seems to be a mismatch between the comment and the code here. Which do we mean?

**Review by ewilkins-csi — APPROVED** [review]

File: extensions/extensions-helm/pom.xml (hunk 1 above)
Comment (chang-annie) [reply extensions/extensions-helm/pom.xml:140]: I think this should be set to true right?

**Review by aaron-gary — APPROVED** [review]

**Review by chang-annie — APPROVED** [review]
looks good! just need to address comment regarding `skipInit`

File: extensions/extensions-helm/pom.xml (hunk 1 above)
Comment (d-ryan-ashcraft) [reply extensions/extensions-helm/pom.xml:139]: Yep - my bad.  We DO need the init, but it needs to NOT have ghcri.io.  Will remove the comment.

File: extensions/extensions-helm/pom.xml [hunk 2]
```diff
@@ -130,10 +156,7 @@
                         </uploadRepoSnapshot>
                         <lintStrict>true</lintStrict>
                         <helmExtraRepos>
-                            <helmRepo>
-                                <name>${aissemble.helm.repo.name}</name>
-                                <url>${aissemble.helm.repo}</url>
-                            </helmRepo>
+                            <!-- No ghcr.io repo here b/c it is oci:// -->
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply extensions/extensions-helm/pom.xml:158]: No - you cannot add an oci repo - instead, you fully quality the link.  This has to do w/ OCI being a direct link to a specific file (no yaml descriptor or search functionality).

**Pull request merged**